packages/renovate-config-cozy/package.json

```diff
@@ -5,7 +5,8 @@
   "renovate-config": {
     "default": {
       "extends": [
-        "config:base"
+        "config:base",
+        ":semanticCommitTypeAll(fix)"
```

Comment:
I am wondering if this needs to be applied only for lib (not for app). What do you think?

Reply:
I don't have strong opinion on it. Let's use it every where and see what happens next?